test/viewUtils/ReportingFrequencyViewUtilsSpec.scala

```diff
@@ -59,10 +78,9 @@ class ReportingFrequencyViewUtilsSpec extends UnitSpec with FeatureSwitching wit
   val reportingFrequencyViewUtils = new ReportingFrequencyViewUtils()
 
   "ReportingFrequencyViewUtils" when {
-
     ".itsaStatusString()" when {
-
       "the ITSAStatus == Mandated" should {
+        disable(OptInOptOutContentUpdateR17)
```

Comment:
can see we've added when the  OptInOptOutContentUpdateR17 but think we need tests for when this is enabled? 

since there is new logic for the new content for when it is enabled.

Reply:
still needs tests to check the content for when the r17 feature switch is on.